src/narrativelog/message.py

```diff
@@ -78,16 +92,31 @@ class Message(BaseModel):
     time_lost_type: None | str = Field(
         title="Type of time lost.",
     )
+    # Added 2024-12-16
+    components_json: None | dict = Field(
+        default_factory=dict,
+        title="JSON representation of systems and subsystems on the OBS jira project. "
+        "For a full list of valid keys please refer to: "
+        "https://rubinobs.atlassian.net/wiki/spaces/LSSTCOM/pages/53741849"
```

Comment:
When I look at the documented linked to for "full list of valid keys", I don't see what those keys are.  When I search that document for "key", the only two matches are "Keysight".   I'm expecting to see something that clearly describes a dictionary (possibly with and example of the full dictionary).

Reply:
Got it :+1:, I'll iterate over this too to make it more specific.